packages/styles/combobox.css

```diff
@@ -0,0 +1,246 @@
+:root {
+  --combobox-input-border-color: var(--field-border-color);
+  --combobox-input-background-color: var(--field-background-color);
+  --combobox-input-border-focus-color: var(--field-border-color-focus);
+  --combobox-input-border-hover-color: var(--field-border-color-focus-hover);
+  --combobox-input-border-focus-glow-color: var(
+    --field-border-color-focus-glow
+  );
+  --combobox-input-error-border-color: var(--field-border-color-error);
+  --combobox-input-error-focus-border-color: var(
+    --field-border-color-error-hover
+  );
+  --combobox-input-error-focus-border-glow-color: var(
+    --field-border-color-focus-glow
+  );
+  --combobox-input-font-size: var(--field-font-size);
+  --combobox-input-font-color: var(--field-content-color);
+  --combobox-listbox-border-color: var(--gray-40);
+  --combobox-listbox-background-color: #fff;
+  --combobox-option-font-color: var(--gray-90);
+  --combobox-option-hover-background-color: var(--gray-20);
+  --combobox-option-description-font-color: var(--gray-60);
+  --combobox-option-description-font-size: var(--text-size-smaller);
+  --combobox-option-min-height: 2.625rem;
+  --combobox-group-label-background-color: var(--gray-20);
+  --combobox-group-label-font-size: var(--text-size-smaller);
+  --combobox-option-selected-icon-color: var(--accent-success-dark);
+}
+
+.cauldron--theme-dark {
```

Comment:
Haven't been in `Cauldron` land for a while, why are we assigning the same values for many of the variables under `theme-dark`?

Reply:
Good question!

This one trips people up because the global var gets set based on its inherited scope. Take `--field-border-color` for example. If we do not set the same value again for `.cauldron-theme-dark`, the inherited value will come from `:root` and not from `.cauldron-theme-dark`, so we need to redefine it again for the value to cascade properly.

Reply:
Is this something like best practice? Whether redefine it in `.cauldron-theme-dark` or not, the value will be the same, right?

Reply:
Best practice would be for it to not be defined twice, but based on how we're inheriting the values here it has to be defined twice. Let me see if I can explain better:

Take the following markup:

```html
<div class="theme">
  <div class="component">Here is some text</div>
</div>
```

...and css:

```css
:root {
  --theme-color: red;
}

.theme {
  --theme-color: green;
}

:root {
  --inherited-theme-color: var(--theme-color);
}

.component {
  color: var(--inherited-theme-color);
}
```

You might think that the color of the text should be green, but it's actually red in this instance.

When the value of `--inherited-theme-color` is getting set, it is using the value of `:root` based on the scope of the current css rule. By the time we reach `.component`, it doesn't know that the variable has been redefined under a different scope because it's using the value from `--inherited-theme-color` which is currently `red`. If we add:

```css
.theme {
  --inherited-theme-color: var(--theme-color);
}
```

...the `--inherited-theme-color` gets set based on the current value for that scope, and correctly displays the value `green`.